packages/create-ponder/src/index.ts

```diff
@@ -159,12 +165,14 @@ export const run = async (
       "scripts": {
         "dev": "ponder dev",
         "start": "ponder start",
+        ${options.eslint ? `"lint": "eslint .",` : ""}
```

Comment:
Oh this is great! I'd add this lint script to all of the examples/ projects too. In general I think it's nice for the examples/ projects to look like they were just generated using create-ponder.